t/cli/test_upstream_mtls.sh

```diff
@@ -0,0 +1,161 @@
+        }
+    }
+}'
+
+sleep 0.1
+
+code=$(curl -v -k -i -m 20 -o /dev/null -s -w %{http_code} http://127.0.0.1:9080/hello)
+
+if [ ! $code -eq 200 ]; then
+    echo "failed: connection to upstream with mtTLS failed"
+    exit 1
+fi
+
+sleep 0.1
+
+make stop
+
+echo "passed: connection to upstream with mtTLS failed success"
+
+# test proxy_ssl_trusted_certificate and use incorrect ca cert
+git checkout conf/config.yaml
```

Comment:
There is no need to check out it as we will write to it soon.

Reply:
rm it